cairo/tests/utils/serde.py

```diff
@@ -254,6 +254,37 @@ def serialize_type(self, path: Tuple[str, ...], ptr) -> Any:
             )
             return Bytes256(data)
 
+        if python_cls is Stack:
```

Comment:
see comment in `arg_gen`, I'd prefer a generic approach where if the `python_cls` is the given `Protocol`, then

```
raw = self._serialize(...)
return python_cls(raw).into()
```